package.json

```diff
@@ -39,7 +39,10 @@
       "baseURL": "src",
       "lib": "src/classmentors"
     },
-    "dependencies": {},
+    "dependencies": {
+      "c3": "npm:c3@^0.4.11",
+      "d3": "npm:d3@^4.2.2"
```

Comment:
It seems to be working on my end Damien. Are you sure that c3's dependency is version specific?